GOVERNANCE.md

```diff
@@ -0,0 +1,356 @@
+
+Users who continue to engage with the project and its community will often
+become more and more involved. Such Users may find themselves becoming
+Contributors, as described in the next section.
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project,
+most often in the form of code and/or documentation. Anyone can become a Contributor,
+and contributions can take many forms, e.g.:
+* Help other users through any of the [communication channels](#communication-channels)
+made for that purpose.
+* Triage GitHub issues.
+* Organize Selenium meetups.
+* Organize and collaborate in Selenium Conferences.
+
+There is no expectation of commitment to the project, no specific skill requirements,
+and no selection process.
+
+Some Contributors might have some basic privileges to the GitHub repos, based on
+their type of contribution (e.g., close an issue after triaging it).
+
+Contributors have read-only access to source code and submit changes via pull
+requests. Contributor pull requests have their contribution reviewed and merged
+by a TLC member or a Committer. TLC members and Committers work with Contributors
+to review their code and prepare it for merging.
+
+As Contributors gain experience and familiarity with the project, their profile within,
+and commitment to, the community will increase. At some stage, they may find themselves
+being nominated for the Committer role by an existing Committer or TLC member.
+
+### Project Committers
+
+Committers are community members who have shown that they are committed to the
+continued development of the project through ongoing engagement with the community.
+Committers are given push access to the project's GitHub repos where they
+contribute to.
+
+Committers:
+
+* Should bring their proposals for large changes to the project's code first to
+a GitHub issue, and all relevant committers should be pinged so they can weigh
+in on the discussion if desired. 
+* Debates between committers about whether code should be merged should happen
+in GitHub pull requests to keep the project decision history.
+* In general any committer can review and merge a pull request. Committers
+should only merge code they are qualified to review, which might entail pinging
+another committer who has greater ownership over a specific code area.
+* May label and close issues.
+
+To become a Committer:
+
+* One must have shown a willingness and ability to participate in the project as
+a team player.
+* Committers are expected to be respectful of every community member and to work
+collaboratively in the spirit of inclusion.
+* Have submitted a minimum of 10 qualifying pull requests. Where significant
+technical weight is present and requires little effort to accept because it is
+well documented and tested.
+
+New Committers can be nominated by any existing Committer or TLC member. Once
+they have been nominated, there will be a vote by the TLC members.
```

Comment:
There seems to be a slight contradiction here in that below it's described that we use consensus-seeking and only use votes if we can't reach consensus, but this says we use a vote. May be worth rephrasing to say that we use the standard consensus-seeking mechanism, rather than a vote?

Reply:
This is a very good point. In reality, many times we say we are voting but we are actually seeking a consensus. I will change the sections that mention `vote` and use `consensus` instead. We do not want situations where someone is added to a committee because the votation went 5 yes and 4 no.

Reply:
I pushed a change with this, please have a look. Hopefully it looks better now.